myhoard/controller.py

```diff
@@ -875,7 +875,7 @@ def _current_normalized_backup_timestamp(self) -> str:
 
         previous_normalized = self._previous_normalized_backup_timestamp()
         # in case we have a previous backup time, we can use it as starting point
-        if previous_normalized:
+        if previous_normalized and backup_interval_minutes > 1440:
```

Comment:
```suggestion
        if previous_normalized and backup_interval_minutes != 1440:
```

And this magic should probably be a magic value from somewhere, but that can be a follow up I guess. Also `>` is good enough for now I think.